learningequality/studio
# Fixes async tasks

## Description

* Does cleanup and better management of sync endpoint polling and error states
* Fixes create_async_task invocation to use ** for task args
* Makes sync nodes non-blocking
* Adds tests for sync nodes assessment item behaviour
* Prevents sync nodes from behaving destructively for unchanged assessment items
* Moves HTML5 thumbnail generation to the client side
* Cleans up unused tasks and other unused backend code
* Cleans up unused dependencies.

#### Issue Addressed (if applicable)

* Fixes #2116
* Fixes #2382
* Fixes #2456

## Steps to Test

- Make a change to a channel
- Publish a channel
- See that channel publishes
- Press refresh

- Sync channel
- This will happen in the background (not sure how we want to track it)

- Generate a thumbnail for an HTML5 content
